packages/itwin/breakdown-trees/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@bentley/breakdown-trees-react",
-  "version": "0.2.0",
+  "name": "@itwin/breakdown-trees-react",
```

Comment:
url on L12 needs to be updated

Reply:
are you guys going to be delivering cjs & esm like all the other pkgs?

Reply:
Thanks for raising the concern. Just fixed our build scripts and pushed a commit to generate cjs and esm.